bazel/README.md

```diff
@@ -406,11 +406,15 @@ be installed.
 
 ```
 bazel build -c dbg //test/common/http:async_client_impl_test
+bazel build //test/common/http:async_client_impl_test.dwp
```

Comment:
This need to be `bazel build -c dbg //test/common/http:async_client_impl_test //test/common/http:async_client_impl_test.dwp`
building dwp without `-c dbg` will result empty dwp file.

Reply:
Now `-c dbg` has been added